cibyl/models/openstack/deployment.py

```diff
@@ -54,6 +54,7 @@ class Deployment(Model):
         }
     }
 
+    # pylint: disable=unsubscriptable-object
```

Comment:
This is fine to keep pylint happy, but it will still fail if run with py36. A type hint like `list[object]` was only supported from  py38+. See [the last-builds PR](https://github.com/rhos-infra/cibyl/pull/62/files#diff-53aaa5335886f4491bcbe2f1dcb49cb054ddc5f02153802d66c1765b279c8e17) where I fixed this too.